src/test/java/com/ipiecoles/java/java350/model/EmployeTest.java

```diff
@@ -66,4 +70,127 @@ public void testNombreAnneeAncienneteNull(){
         Assertions.assertThat(nbAnneesAnciennete).isEqualTo(0);
     }
 
+    @Test
+    //le pourcentage est t-il présent
+    public void testAugmenterSalairePourcentagePresent(){
+        //Given
+         Employe employe = new Employe();
+         Double pourcentage = 0d;
+
+         //When
+        employe.augmenterSalaire(pourcentage);
+
+         //Then
+        Assertions.assertThat(pourcentage).isNotEqualTo(null);
+    }
+
+
+    @Test
+    //test pourcentage supérieur à 1
+    public void testAugmenterSalaireInf(){
+        //Given
+        Double pourcentage = 2d;
+        Employe employe = new Employe();
+        employe.setSalaire(1000d);
+        Double resultat = employe.getSalaire();
+
+        //When
+        employe.augmenterSalaire(pourcentage);
+
+        //Then
+        Assertions.assertThat(employe.getSalaire()).isGreaterThan(1);
```

Comment:
Utilise des Assertions précises ! Car si je modifie le code d'augmenterSalaire pour dire que je rajoute un euro sur ton salaire que je souhaite augmenté de 2 % ton test passera et pourtant tu vas te sentir roulé...
```suggestion
        Assertions.assertThat(employe.getSalaire()).isEqualTo(1020d);
```